.github/workflows/ci.yml

```diff
@@ -304,3 +306,33 @@ jobs:
         with:
           config: '.markdownlint.yaml'
           globs: '**/README.md'
+
+  build_shared:
+    name: Build shared libs
+    uses: ./.github/workflows/build-shared.yaml
+
+  build_static:
+    name: Build shared libs
```

Comment:
```suggestion
    name: Build static libs
```

Reply:
Fixed in https://github.com/eclipse-zenoh/zenoh-pico/pull/833/commits/f074018aada41c11ef7a4658213aad62384b9e15